package.json

```diff
@@ -26,15 +26,15 @@
     "@mediapipe/tasks-vision": "0.10.9"
   },
   "peerDependencies": {
-    "livekit-client": "^1.12.0 || ^2.0.0"
+    "livekit-client": "^1.12.0 || ^2.1.0"
   },
   "devDependencies": {
     "@changesets/cli": "^2.26.2",
     "@livekit/changesets-changelog-github": "^0.0.4",
     "@trivago/prettier-plugin-sort-imports": "^4.1.1",
     "@types/dom-mediacapture-transform": "^0.1.6",
     "@types/offscreencanvas": "^2019.7.0",
-    "@typescript-eslint/eslint-plugin": "^4.31.2",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
```

Comment:
This was the minimal version I saw that satisfied peer dependency conflicts after updating livekit-client, above. There are quite a few more recent versions of this package than this.